code/modules/cm_aliens/weeds.dm

```diff
@@ -481,9 +481,12 @@
 	flags_atom = OPENCONTAINER
 	layer = ABOVE_BLOOD_LAYER
 	plane = FLOOR_PLANE
-	var/static/staticnode
+	var/staticnode
```

Comment:
Poor static type. We hate static =D
But are u sure about it? U dont want common parameters everywhere? It could negatively affect everything...
Is one structure really worth changing everything?

Reply:
changed that to different variable name, it cannot be static because it will fail replacement.